Eastern-Research-Group/expert-query
# Feature/210 update dependencies

## Related Issues:
* https://jira.epa.gov/browse/EQ-210

## Main Changes:
* Updated dependencies.

## Steps To Test:
1. Run `npm ci` at the etl, app, and app/server levels
2. Run `npm ci --legacy-peer-deps` at the app/client level
3. Run the etl and verify it succeeds
4. Run the app
5. Test as many features as possible

## TODO:
- [ ] Fix the cypress tests. I'll create a ticket for this.
    * I started to work on this, but since we are going to be changing up all of the api endpoints, I decided to hold off on this.
- [ ] Do another round of dependency updates prior to releasing for codescan. I'll create a ticket for this.
